curriculum/challenges/english/01-responsive-web-design/responsive-web-design-projects/build-a-tribute-page.md

```diff
@@ -162,12 +159,13 @@ Your `#image` should have a `height` of `auto`
 ```js
 // taken from the testable-projects repo
 const img = document.getElementById('image');
-const maxWidthValue = new __helpers.CSSHelp(document).getStyle('#image')?.getPropertyValue('max-width')
-const displayValue = new __helpers.CSSHelp(document).getStyle('#image')?.getPropertyValue('display')
+const imgStyle = window.getComputedStyle(img);
+const maxWidthValue = imgStyle?.getPropertyValue('max-width')
+const displayValue = imgStyle?.getPropertyValue('display')
```

Comment:
Why are these values defined here if we are only checking for the height property?

Reply:
Because the other test suite uses them so they *must* be important right? 🙃